### ISSUE TITLE
issues pip isntalling in colab (samplerate)

### ISSUE THREAD

**Issue by ajayarora1235** [karaokenerds/python-audio-separator#40]
Getting this error when pip installing in colab:

```
Building wheels for collected packages: samplerate
  error: subprocess-exited-with-error
  
  × Building wheel for samplerate (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> See above for output.
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  Building wheel for samplerate (pyproject.toml) ... error
  ERROR: Failed building wheel for samplerate
Failed to build samplerate
ERROR: Could not build wheels for samplerate, which is required to install pyproject.toml-based projects
```

And when I try 0.14.3, I'm unable to use any of the VR models as this error comes up, even though Colab runs an Ubuntu setup.

**Comment by GoombaFCU** [comment]
I've encountered the same issue on Ubuntu. My solution is to install 0.1.0 version of samplerate, and it working properly so far.

**Comment by beveradb** [comment]
Should be fixed in version `0.14.5`, I pinned that older version of samplerate which doesn't require cmake to build: https://github.com/karaokenerds/python-audio-separator/releases/tag/v0.14.5
